sh_files/ressources/.devrc

```diff
@@ -9,7 +9,15 @@ if [ -r "$HOME/.bashrc" ]; then
     . "$HOME/.bashrc"
 fi
 
+
+# Fonction qui permet de changer le titre
+setTitle() {
+    echo -en "\033]0;$@\007" 
+}
+
+alias SARACMD='eval $SARACMD'
```

Comment:
I would prefer RUNSARACMD to differentiate from the environnement variable

Reply:
On pourrait plutot changer le nom de la variable